Sources/Classes/core/JavascriptBridge/MiniAppScriptMessageHandler.swift

```diff
@@ -52,7 +52,7 @@ internal class MiniAppScriptMessageHandler: NSObject, WKScriptMessageHandler {
         handleActionCommand(action: requestAction, requestParam: requestParam, callbackId: callbackId)
     }
 
-    func handleActionCommand(action: MiniAppJSActionCommand, requestParam: RequestParameters?, callbackId: String) {
+    func handleActionCommand(action: MiniAppJSActionCommand, requestParam: RequestParameters?, callbackId: String) { // swiftlint:disable:this function_body_length
```

Comment:
this looks like the swiftlint statement is inside the function body but shouldn't it be on top of the funtion like so?

```suggestion
    // swiftlint:disable:this function_body_length
    func handleActionCommand(action: MiniAppJSActionCommand, requestParam: RequestParameters?, callbackId: String) {
```

Reply:
Yeah, that's how we used to get rid of the warning right? But this one doesn't seem to work that way, when I added that. Now after cleaning up and added the disable command like you said. Its working fine now 🙄

Reply:
😄 the almighty clean & run